docs/static/ds6/docs.js

```diff
@@ -1,22 +1,22 @@
-$_mod.installed("@ebay/skin$6.0.4","makeup-expander","0.5.0");
-$_mod.main("/makeup-expander$0.5.0","");
-$_mod.installed("makeup-expander$0.5.0","custom-event-polyfill","0.3.0");
+$_mod.installed("@ebay/skin$6.0.4","makeup-expander","0.4.0");
+$_mod.main("/makeup-expander$0.4.0","");
```

Comment:
You'll need to run yarn I think to make sure latest version of expander is pulled in. Assuming you rebased the recent devDeps update.

Reply:
Rebased.

Reply:
Okay, rebuilt and should be correct now.